release-notes-1.2.0.md

```diff
@@ -0,0 +1,761 @@
+
+```ballerina
+import ballerina/lang.'xml as xmllib;
+
+public function main() {
+     xmllib:Element element = <xmllib:Element> xml `<elem> hello </elem>`;
+     xml children = element.getChildren();
+   }
+```
+
+#### The `lang.boolean` module
+
+The newly-added lang library module for the `boolean` basic type contains a function for parsing `string` values to `boolean` values. It accepts `”true”` or `”false”` in any combination of lower/upper case as well as `”1”` and `”0”`, which evaluates to `true` and `false` respectively. An error is returned for any other `string` value.
+
+```ballerina
+import ballerina/lang.'boolean;
+
+public function main() {
+    boolean|error b = 'boolean:fromString("true");
+}
+```
+
+#### Enhanced `lang.int`, `lang.string`, `lang.array`, and `lang.map` modules
+
+The above-mentioned lang library modules have been enhanced by adding new API elements (e.g., constants and functions). 
+
+- `lang.int` module - Constants added: for max/min values of the `int` type 
+- `lang.string` module - New functions added: `lastIndexOf()`, `equalsIgnoreCaseAscii()`, `toCodePointInt()`, `fromCodePointInt()`
+- `lang.array` module - New functions added: `lastIndexOf()`, `toStream()`
+- `lang.map` module - New functions added: `toArray()`, `removeIfHasKey()`
+
+## Runtime
+
+### Compilation time improvement
+
+Major improvements to the compilation (code-generation) time are implemented. The compiler backend, which generates the Java bytecode for Ballerina programs is now rewritten to improve the overall compilation time. This change has specifically reduced the total time it takes for Java bytecode generation.
+
+### The `@strand` annotation
+
+As per the Ballerina language specification, new strands should belong to the same thread as the currently executing strand. Strands, which belong to the same thread share the following characteristics. 
+
+- Only one strand can run simultaneously.
+- The currently-running strand should yield to switch its thread to execute another strand.
+- Strands belonging to a particular thread are cooperatively multitasked.
+- All the strands created in a particular execution of a Ballerina program will be executed sequentially by a single thread. 
+
+In some situations, it would be desirable to assign strands to different threads. You can change this behaviour with an annotation but such an annotation was not available in Ballerina releases prior to 1.2.0. This has been a major spec deviation. 
+
+A major revamp of our strand scheduler was done in this release to address the spec deviation. 
+
+## Standard Library
+
+### Enhanced HTTP module
+
+#### Cookie support
+
+- Enable the server to read cookies from the request and write cookies to the response (in "Set-Cookie" header).
```

Comment:
This needs to be a complete sentence

Reply:
Same problem in the two sentences below as well

Reply:
This is already a complete sentence. @pubudu91 - Can you edit your suggestion here itself?

Reply:
A sentence was wrong word for it. I meant, As a whole, in this context it's not really correct. It breaks off from the format in which the other content are written. Will see if I can come up with something